frappe/desk/reportview.py

```diff
@@ -436,10 +434,18 @@ def get_labels(fields, doctype):
 	"""get column labels based on column names"""
 	labels = []
 	for key in fields:
-		key = key.split(" as ")[0]
+		aggregate_function = ""
+
+		for sep in (" as ", " AS "):
+			if sep in key:
+				key = key.split(sep)[0]
```

Comment:
```suggestion
		if " as " in key.casefold():
			key = key.split(sep)[0]
```

Reply:
It will not work since **sep** can still be uppercase or lowercase😅
```suggestion
		for sep in (" as ", " AS "):
			key = re.sub(' as ', ' AS ', key, flags=re.IGNORECASE)
			key = key.split(" AS ", 1)[0]
```